### DISCUSSION TITLE
How to recover funds after a hardware failure?

### DISCUSSION THREAD

**Discussion by gittbuh** [raspiblitz/raspiblitz#4203] [Q&A]
My node had a hardware failure about 5 moths ago. 

I am planning now to re-install Raspiblitz in a VM in Proxmox and wonder if and how I can recover my funds. 

My previous installation was storing some backups on my Nextcloud. These are 2 files called channel-xxx-xxx.backup and 1 file called channel.backup. Can I recover my funds from these files within my new installation?

Thank you.

**Comment by adevoss** [comment]
If the HDD is OK, the first option would be just to reinstall Raspiblitz.

If using a Raspberry Pi then create a new SD card.
If you're using Proxmox (like me) then install Debian 11(!) and sudo and then download/execute build_sdcard.sh with the options headless, no wifi, no tweaking harddisk (maybe I'm forgot to mention an option).

I went for v1.10 but that is up to you. It's not a release (yet).

**Reply by gittbuh** [reply]
Thanks, I am afraid the hard disk might be the problem. If that was the case, what next?

**Comment by adevoss** [comment]
Restore from backup I think. Luckily I have no experience how to do that.
`blitz.migration.sh export`
`blitz.migration.sh import`
